File: src/java/Figma.java
package io.quicktype;

import java.util.Map;
import com.fasterxml.jackson.annotation.*;

/**
 * A geographical coordinate
 */
public class Figma {
    private Double latitude;
    private Double longitude;

    @JsonProperty("latitude")
    public Double getLatitude() { return latitude; }
    @JsonProperty("latitude")
    public void setLatitude(Double value) { this.latitude = value; }

    @JsonProperty("longitude")
    public Double getLongitude() { return longitude; }
    @JsonProperty("longitude")
    public void setLongitude(Double value) { this.longitude = value; }
}

File: src/go/figma.go
// To parse and unparse this JSON data, add this code to your project and do:
//
//    figma, err := UnmarshalFigma(bytes)
//    bytes, err = figma.Marshal()

package main

import "encoding/json"

func UnmarshalFigma(data []byte) (Figma, error) {
	var r Figma
	err := json.Unmarshal(data, &r)
	return r, err
}

func (r *Figma) Marshal() ([]byte, error) {
	return json.Marshal(r)
}

// A geographical coordinate
type Figma struct {
	Latitude  *float64 `json:"latitude"` 
	Longitude *float64 `json:"longitude"`
}
